### DISCUSSION TITLE
SQL unrecognizing database

### DISCUSSION THREAD

**Discussion by davonnep** [pawelsalawa/sqlitestudio#5121] [General]
<img width="916" alt="Screenshot 2024-11-23 at 9 42 56 AM" src="https://github.com/user-attachments/assets/d508844c-b03f-4a72-8a96-bc9b5d2143df">
### Details
Trying to select all from my database, customers portion, but it keeps giving me an error

### Operating system: apple macOS
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by Sierron** [comment]
The error message you get is pretty clear what the issue is. You need to query an actual table. You can't query the database file itself like this.

Although, attached databases can be queried by the database name that you give in the sql query:

ATTACH DATABASE file_name AS database_name;

and then

SELECT * FROM contacts.people;

**Comment by pawelsalawa** [comment]
This is how you query particular table. You cannot query the database. You have to query a table.
`SELECT * FROM AprSales;`
